util/rom-ext-manifest-generator.py

```diff
@@ -130,14 +174,23 @@ def main():
                         type=Path,
                         help='Manifest header output directory.')
 
+    parser.add_argument('--output-headers',
+                        required=True,
+                        choices=['c', 'rust'],
+                        help='The type of headers to be produced.')
+
     args = parser.parse_args()
 
     manifest_hjson_file = args.input_dir / 'manifest.hjson'
 
     with manifest_hjson_file.open('r') as hjson_file:
         obj = hjson.loads(hjson_file.read())
 
-    generate_cheader(obj['fields'], args.input_dir, args.output_dir)
+    for header in [args.output_headers]:
```

Comment:
This will only ever be a singleton list, I think, so I'm not sure a for loop is quite right - if you want to be able to select both c and rust at the same time, look at how `util/make_new_dif.py` handles the `--only` argument.

Reply:
Done.

Still cant do `--output-headers=c rust`, but is better nevertheless.

Reply:
You should be able to do `--output-headers=c --output-headers=rust`.

Reply:
Yes, It does work.